docs/README.md

```diff
@@ -39,16 +39,15 @@ systems, but all the principles apply equally to such scenarios.
 The image below shows the modules structure of a single SIP adapter.
```

Comment:
The image needs to be updated as well. Currently it shows dependencies between modules etc.